From: Simon McVittie <smcv@debian.org>
Date: Tue, 29 Nov 2016 10:15:16 +0000
Subject: test-run: be a bash script

set -o pipefail is a bashism, and doesn't work in an ordinary POSIX sh;
in particular, /bin/sh in Debian and Ubuntu is usually dash, which
doesn't have set -o.

Signed-off-by: Simon McVittie <smcv@debian.org>
Closes: #120
Approved by: alexlarsson
Applied-upstream: 0.1.5, commit:7c8d421
---
 tests/test-run.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/test-run.sh b/tests/test-run.sh
index 70aee4c..cd31e84 100755
--- a/tests/test-run.sh
+++ b/tests/test-run.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 
 set -xeuo pipefail
 
